general/g.setproj/g.setproj.html

```diff
@@ -85,11 +85,5 @@ <h2>AUTHORS</h2>
 Morten Hulden, morten at untamo.net - rewrote module and added 121 projections <br>
 Andreas Lange, andreas.lange at rhein-main.de - added prelimnary map datum support
 
-<!--
-<p>
-<i>Last changed: $Date$</i>
--->
-
-
 </body>
 </html>
```

Comment:
Seems `</html>` and `</body>` HTML element tags are extra, same `<!DOCTYPE>` declaration at the top of the manual page?

Reply:
This is a leftover module: https://github.com/OSGeo/grass/blob/76cf64cad28dd9cd092a7d8fc6875c71da071eb4/general/Makefile#L30

It doesn't have proper parser support and hence cannot generate the full manual page.
In a separate PR, we could consider to drop `g.setproj` entirely.